balloc/balloc.c

```diff
@@ -394,6 +399,9 @@ static int balloc_group_info_init(struct m0_balloc_group_info *gi,
 				 normal_zone_size,
 				 spare_zone_size, 0, 0, 0);
 #endif
+		M0_LOG(M0_INFO, "cb_db_group_desc loading grpNo : %llu  freeblk : %llu",
```

Comment:
Looks like column width 80 is exceeded in above line.
"The preferred limit on the length of a single line is 80 columns"